yarn-project/noir-compiler/src/compile/noir/dependencies/local-dependency-resolver.ts

```diff
@@ -16,12 +16,14 @@ export class LocalDependencyResolver implements NoirDependencyResolver {
     this.#fm = fm;
   }
 
-  resolveDependency(pkg: NoirPackage, config: NoirDependencyConfig): Promise<NoirDependency | null> {
+  resolveDependency(parent: NoirPackage, config: NoirDependencyConfig): Promise<NoirDependency | null> {
     if ('path' in config) {
+      const parentPath = parent.getPackagePath();
+      const dependencyPath = isAbsolute(config.path) ? config.path : join(parentPath, config.path);
```

Comment:
Kinda worried about someone putting `/etc/passwd` in their dependency list. Maybe we make it only accept relative deps?

Reply:
`resolveDependency` is only used for _loading_ stuff, right? We're not writing to the resolved directory at any time? If so, I think it's not that bad, since at most it'd exfiltrate `/etc/passwd` to the noir compiler, and won't overwrite anything.